apm-agent-core/src/main/java/co/elastic/apm/agent/report/serialize/DslJsonSerializer.java

```diff
@@ -723,7 +723,8 @@ private void serializeDbContext(final Db db) {
                 }
             }
             writeField("type", db.getType());
-            writeLastField("user", db.getUser());
+            writeField("user", db.getUser());
+            writeLastField("link", db.getDbLink());
```

Comment:
`writeLastField` always adds a field to the serialized JSON, even if the value is null, so such a field (I assume not often used) is not the best fit. Better switch the order and make the `type` the last field.